io/serialport/25-serial.html

```diff
@@ -245,7 +250,8 @@
             bin: {value:"false"},
             out: {value:"char"},
             addchar: {value:""},
-            responsetimeout: {value: 10000}
+            responsetimeout: {value: 10000},
+            serialReconnectTime: {value: 15000}
```

Comment:
Can't set this here - as then settings.js would never be able to override... must default to blank

Reply:
I have updated it, now its an empty string.